.travis.yml

```diff
@@ -187,14 +187,14 @@ jobs:
     env:
     - DOCKER_REPOSITORY=geosx/centos7.6.1810-gcc8.3.1-cuda10.1.243
     - CMAKE_BUILD_TYPE=Release
-    - BUILD_AND_TEST_ARGS="--disable-unit-tests --reduce-install-logs"
+    - BUILD_AND_TEST_ARGS="--disable-unit-tests --reduce-install-logs --disable-schema-deployment"
```

Comment:
Passing by, can you remove `--reduce-install-logs` which looks like it's become pretty useless...